### PR TITLE
Fix the props name passed in ContactViaConnect

### LINKED ISSUE #4336: "Contact via Connect" in listings form cannot be unchecked
The button in the main listings form does not change when the checkbox is clicked...

Expected behavior:

The checkbox changes and the appropriate preact state is altered.

### PR DESCRIPTION
**Pull request by KattyaCuevas**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [X] Bug Fix
- [ ] Documentation Update

## Description
The checkbox in the `ContactViaConnect` component changes correctly and the preact state is updated.

## Related Tickets & Documents

Fixes #4336 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
No screenshots needed

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [X] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

It's happening

### CODE REVIEW

File: app/javascript/listings/listingForm.jsx [hunk 1]
```diff
@@ -119,3 +127,17 @@ export default class ListingForm extends Component {
...
+  categoriesForSelect: PropTypes.string,
+  organizationsString: PropTypes.string,
+};
+
+ListingForm.defaultProps = {
+  listing: '',
+  categoriesForDetails: '',
+  categoriesForSelect: '',
+  organizationsString: '',
+};
```
Comment (rhymes) [inline app/javascript/listings/listingForm.jsx]: shouldn't the props be required? 

I recently reviewed an open PR that's also working on this file and the author there set the props as required https://github.com/thepracticaldev/dev.to/pull/4253/files

File: app/javascript/listings/elements/contactViaConnect.jsx [hunk 2]
```diff
@@ -5,21 +5,21 @@ const ContactViaConnect = ({ onChange, checked }) => (
...
     <label className="listingform__label" htmlFor="contact_via_connect">
       Allow Users to Message Me Via In-App Chat (DEV Connect)
+      <input
+        type="checkbox"
+        className="listingform__input listingform__contact_via_connect"
+        id="contact_via_connect"
+        name="classified_listing[contact_via_connect]"
+        onInput={onChange}
+        checked={checked}
+      />
```
Comment (rhymes) [inline app/javascript/listings/elements/contactViaConnect.jsx]: does this change the UI in any way? moving the input inside the label...

File: app/javascript/listings/listingForm.jsx [hunk 3]
```diff
@@ -84,18 +92,18 @@ export default class ListingForm extends Component {
...
+            listing
           />
           <ExpireDate
             defaultValue={expireDate}
             onChange={linkState(this, 'expireDate')}
           />
           {selectOrg}
           <ContactViaConnect
-            defaultValue={contactViaConnect}
+            checked={contactViaConnect}
```
Comment (rhymes) [inline app/javascript/listings/listingForm.jsx]: this is the actual fix, correct?

**Comment by mstruve** [comment]
Hi @KattyaCuevas!

It looks like the related ticket ended up being fixed by another PR, https://github.com/thepracticaldev/dev.to/pull/4617 so I am going to go ahead and close this one.

Thank you so much for contributing! We couldn't do what we do without our awesome open source contributors 😄

**Pull request closed without merging**